kodadot/nft-gallery
# Club-Mate-Refactor

- fix: format balance warning
- fix: disable submitting before RPC api connected
- fix: refresh page index after changing search keyword
- ✨ test enable extension
- ✨ test preview enable extension
- feat: make 'isApiConnected' be global and handle the error event
- chore: remove unused 'async' keyword
- feat: Cache request from Identity popover
- remove useless import
- fix: move the filter to formatBalance
- feed history on collection page
- add sorting by date
- remove unused types
- #2360 Hoverable card in CarouselCardList component
- [skip ci] updated code diagram
- fix: add format balance test case
- fix: code format
- subsquid query + default selectEvent to BUY if BUY events?
- Update components/rmrk/Gallery/CollectionItem.vue
- #1262 Nudge to set on-chain identity on profile
- remove console
- [skip netlify] Update dependency eslint to ^8.11.0
- Add timestamp on events + util sort event by date + create other issues for default selectedEvent on History.vue
- fix: fix error when estimate fee without connecting wallet
- fix: ignore keyboard event in input
- feat: check network status
- feat: replace navigator.connection with global offline/online event
- fetch identity on mounted()
- refactor: store typing improvement
- use utils/api/onApiConnect instead
- try to fix deepsource by adding vue lifecycle hooks docs
- fix: `isNetworkConnected` default to true
- fix: fixlink unclickable on ios safari
- feedback review
- ✨ UNLIST event in history
- test remove enableExtension
- wip
- how to generate payout address for one self
- updated bug issue template with address field
- quote missing
- :memo: fixing typos through the document and clarifying p1 labels
- ⚒️ linting errors
- Update CONTRIBUTING.md
- Update CONTRIBUTING.md
- Update CONTRIBUTING.md
- fix: ts -lint error
- feat: history component pagination
- [skip netlify] Update Bumped minor version of packages
- :memo: add that we use yarn as the package manager
- Update components/rmrk/Gallery/History.vue
- Update components/rmrk/Gallery/History.vue
- :heavy_plus_sign: added @babel/preset-env
- feat: history component pagination
- feat: show context of page in navbar
- changed to coordination channel )
- update with 12hr cache
- fix: code style justify
- fix: import mock address KODADOT_DAO
- Update History.vue
- fix: add item column and relative time
- fix: change ui
- fix: url query event fail
- fix: amount of mint、gift and consume should be zero
- fix review, use identityOf instead
- chore: remove unnecessary css
- remove "TO" column when there is no data
- refactor code, more robust
- lets test if it change w vuejs/vue-cli
- adding vue/cli-plugin for rennovate
- [skip netlify] Update Bumped minor version of packages
- use date-fns
- fix review, remove console.log on:connect
- [skip ci] updated code diagram
- fix with advice
- fix: showlist instead of data
- feat: add ability to download asset
- fix: add regex to clean item id and dont show on gallery
- [skip netlify] Update dependency @kodadot1/minimark to ^0.0.1-rc.6
- fix: export name
- fix: change file name
- feat: add link to collection history
- fix: correct name of variable
- #2539 Post fixes in sales feed of collection
- #2549 Made search bar more transparent and bit wider
- Update components/rmrk/Gallery/History.vue
- added pretttier and reorg rennovate
- fix: use totalcount for vercel rendering
- fix: code refactor
- 🧪 remove blocking enableExtension
- fix: justifysize
- 🧪 revert some change
- 🧪 reset walletAccount on setWallet
- fix: remove bug action when it is not for sale
- #2549 Search issue fix
- 🧪 hide options if empty
- fix: scrolling do not work
- 🧪 Object assign walletAccounts
- 🧪 handle watch walletAccounts
- #2578 Keep only top-border on pagination and sharing buttons
- css fix
- global css fix
- 🧪 test test un deux walletAccounts
- lint
- 🧹 css stuff
- [skip netlify] Update Bumped minor version of packages
- #2582 Pagination on history collection component fix on mobile
- fix: keyboard shortcut on creat page
- add name and id in collection history
- maybe @babel/core #2588
- connect 1px border
- feel desperate #2588
- chaning navbar to 1px
- [skip netlify] Update dependency @babel/core to ^7.17.8
- fix review, proper type for TableRowItem
- fix: Sharing search url with page number is broken
- fix: searchbar at index page wont redirect to search result
- Revert "fix: refresh page index after changing search keyword"
- fix: refresh page index after searching new keyword
- fix: update url page
- fix: ts error
- fix: collection list url share
- resolve conflicts
- revert github diagram.svg
- fix: tile view does not work after entering theatre view

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type


### What's new?


### Before submitting Pull Request, please make sure:


### Optional


### Had issue bounty label?


### Community participation


### Screenshot
